casper/src/main/rholang/MakeBrandPair.rho

```diff
@@ -0,0 +1,29 @@
+//scalapackage coop.rchain.rholang.security
+
+//Based on sealer/unsealer pattern in http://erights.org/elib/capability/ode/ode-capabilities.html
+//
+//This design pattern makes it possible to restrict *access* to data/methods, but
+//not necessarily *knowledge* of the data/methods (because everything is public 
+//on a blockchain). However, restricting access is enough to ensure security in
+//some situations. For example one could seal the method on a "purse" object which
+//extracts the tokens inside, thus allowing only those with the unsealer to 
+//take coins out of the purse.
+contract @"MakeBrandPair"(return) = {
+  new seal, unseal, key in {
+    contract seal(@data, return) = {
+      new sealedData in {
+        contract sealedData(@input, return) = {
+          bundle+{
```

Comment:
I don't know what you're trying to achieve with this bundle. It doesn't seem to accomplish anything.

Reply:
It seems that what you might want to do instead is to send a list/tuple where the first element is the name, and the second is a bundle0 that uses that name in a linear for.